README.md

```diff
@@ -1,121 +1,26 @@
-# dig: Dependency Injection Framework for Go
+# :hammer: dig [![GoDoc][doc-img]][doc] [![Build Status][ci-img]][ci] [![Coverage Status][cov-img]][cov] [![Report Card][report-card-img]][report-card]
```

Comment:
Obviously would love to switch the hammer out for some type of hand shovel:

https://www.iconfinder.com/search/?q=shovel

For the time being, the hammer works well with the idea that dig is a toolkit, not an application framework to be used directly.